go/libraries/doltcore/sqle/dolt_session.go

```diff
@@ -115,20 +115,19 @@ func TableCacheFromSess(sess sql.Session, dbName string) TableCache {
 	return sess.(*DoltSession).caches[dbName]
 }
 
-func (sess *DoltSession) CommitTransaction(ctx *sql.Context) error {
-	currentDb := sess.GetCurrentDatabase()
-	if currentDb == "" {
-		return sql.ErrNoDatabaseSelected.New()
+func (sess *DoltSession) CommitTransaction(ctx *sql.Context, dbName string) error {
+	if dbName == "" {
```

Comment:
When does this happen?

Reply:
Leave a comment

Reply:
This was a bug that was happening when dbeaver would run a query and throw a "commit" statement when no database was selected. For some reason even with autocommit off, dbeaver forces these transaction statements on you.